common-rendering/src/components/liveBlockContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@emotion/react';
+
+import { from } from '@guardian/src-foundations/mq';
+import { neutral, space } from '@guardian/src-foundations';
+
+const LEFT_MARGIN_DESKTOP = 60;
+const SIDE_MARGIN = space[5];
+const SIDE_MARGIN_MOBILE = 10;
+
+const LiveBlockContainer = ({
+	id,
+	children,
+	borderColour,
+}: {
+	id: string;
+	children: React.ReactNode;
+	borderColour: string;
+}) => {
+
+	return (
+		<div
+			id={`block-${id}`}
+			css={css`
+				padding: ${space[2]}px ${SIDE_MARGIN_MOBILE}px;
+				margin-bottom: ${space[3]}px;
+				background: ${neutral[100]};
+				border-top: 1px solid ${borderColour};
+				${from.tablet} {
+					padding: ${space[2]}px ${SIDE_MARGIN}px;
+					padding-left: ${LEFT_MARGIN_DESKTOP}px;
+				}
+			`}
+		>
+			<main>{children}</main>
+		</div>
```

Comment:
Just thinking about semantics, should this be?

```tsx
<article id="..." css={...}>
  {children}
</article>
```

I don't think you're allowed to have more than one `<main>` in a document: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main

However, multiple `<article>`s is fine, and I think is semantically exactly what we want here: each block is basically a standalone "article". I think we may also be able to avoid the wrapping `div` if we can style on the `<article>` tag directly?

Reply:
Yeah, good shout. Thanks Jamie 😄